examples/json/source/app.d

```diff
@@ -17,4 +17,12 @@ void main()
 	Json parent = obj;
 	parent.remove("item1");
 	foreach (i; obj) writeln(i);
+
+	version (VibeJsonDisableBigInt) {
+	} else {
+		import std.bigint;
+		auto obj2 = parseJsonString(`{"serial":17559991181826658461}`);
+		writeln("serial: ", obj2["serial"]);
+		assert(obj2["serial"] == BigInt(17559991181826658461UL));

```

Comment:
Also, it'd be good to have a test that test number outside of a size_t boundary, to ensure that this part is also working.